app/managers.py

```diff
@@ -0,0 +1,33 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.connection_ = sqlite3.connect("cinema")
+        self.table_name = "actors"
+
+    def all(self) -> list:
```

Comment:
Your function returns list of Actor. You should show it in type hint
```suggestion
    def all(self) -> list[Actor]:
```